tests/phpunit/WC_Stripe_Webhook_Handler_Test.php

```
@@ -884,4 +889,238 @@ public function provide_test_process_webhook_refund_updated() {
+		$prop->setValue( $this->mock_webhook_handler, $mock_scheduler );
+
+		$this->mock_webhook_handler->process_checkout_session( $notification );
+
+		// Verify the metadata contains the correct order data.
+		$this->assertNotNull( $scheduled_args );
+		$this->assertEquals( $checkout_session_id, $scheduled_args['checkout_session_id'] );
+		$this->assertEquals( $order->get_order_number(), $scheduled_args['metadata']['order_id'] );
+		$this->assertEquals( $order->get_order_key(), $scheduled_args['metadata']['order_key'] );
+		$this->assertNotEmpty( $scheduled_args['metadata']['signature'] );
+		$this->assertIsInt( $scheduled_args['metadata']['tax_amount'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_process_checkout_session_schedules_metadata_job is not named in camelCase. (undefined)

(CamelCaseMethodName)